ingest/muxer.go

```diff
@@ -1439,7 +1452,7 @@ func (eq *emergencyQueue) clear(igst *IngestConnection, tt *tagTrans) (ok bool)
 					if !ok {
 						// could not translate, push it back on the queue and bail
 						// first we need to reverse the ones we have already translated, ugh
-						for j := 0; j < i; j++ {
+						for j := 0; j <= i; j++ {
```

Comment:
This will attempt to reverse a tag which never got translated in the first place... I think j < i was correct.